### PR TITLE
allow the creation of tree from descendant data

### PR DESCRIPTION
**Pull request by jobo322**

### CODE REVIEW

File: src/createTree.js [hunk 1]
```diff
@@ -11,23 +11,32 @@ import {asc} from 'num-sort';
...
         minWindow = 0.16,
         threshold = 0.01,
         from = X[0],
         to = X[X.length - 1]
     } = options;
 
-    return mainCreateTree(spectrum[0], spectrum[1], from, to, minWindow, threshold);
+    if (X[0] > X[1]) {
+        if (from < to) [from, to] = [to, from];
+    } else if (from > to) {
```
Comment (maasencioh) [inline src/createTree.js]: everything can be written in a single condition, was splited with any purpose?

**Review by maasencioh — COMMENTED** [review]
@jobo322 I didn't understand what did you mean with `create tree with descendant data`

File: src/createTree.js (hunk 1 above)
Comment (jobo322) [reply src/createTree.js]: I mean with descendant data when the delta of x axis is negative, before, the package return null if the argument from > to or 'deltaX' < 0 and usually the NMR data has 'deltaX' < 0. It change is just for be able to create the tree without reverse the data. Now, is it ok for you? or do you prefer reverse the data.

Comment (maasencioh) [reply src/createTree.js]: no, it's ok for me, I just didn't understand the idea :+1:

**Pull request closed without merging**